day3-b.js

```diff
@@ -0,0 +1,33 @@
+//**Hello Ramin this is part 1 of day 3 :D **//
+//i am running this online to get the input here : https://www.codechef.com/ide //
+process.stdin.resume();
+process.stdin.setEncoding('utf8');
+
+
+process.stdin.on('data', function(data) { 
```

Comment:
Always use the opportunity to name your functions. As a reviewer I cannot easily understand what this function tries to achieve.

```suggestion
process.stdin.on('data', function getPowerConsumption(data) { 
```